### PR TITLE
Issue #1507 Event title with special chars in Google calendar

### PR DESCRIPTION
**Pull request by anthonykgross**
Work in progress
To do : tests

### CODE REVIEW

**Comment by anthonykgross** [comment]
Issue #1507 fixed. 
New event created in the tests fixtures (Event_14).

File: src/DataFixtures/ORM/LoadEventData.php [hunk 1]
```diff
@@ -252,6 +253,21 @@ public function load(ObjectManager $manager)
...
+            'committee' => $committee9,
+            'name' => 'Meeting #11 de Brooklyn',
+            'category' => $eventCategory10,
+            'description' => 'Ouvert aux français de New York.',
+            'address' => PostAddress::createForeignAddress('US', '10019', 'New York', '226 W 52nd St', 40.7625289, -73.9859927),
+            'begin_at' => (new \DateTime('now', new \DateTimeZone('America/New_York')))->modify('+10 hours')->format('Y-m-d H:00:00'),
+            'finish_at' => (new \DateTime('now', new \DateTimeZone('America/New_York')))->modify('+13 hours')->format('Y-m-d H:00:00'),
+            'capacity' => 55,
+        ]);
+        //
```
Comment (illusionOfParadise) [inline src/DataFixtures/ORM/LoadEventData.php]: Extra slashes

**Comment by illusionOfParadise** [comment]
@anthonykgross thanks for the contribution. 
I added a few small comments.
Please also rebase your branch to update with last version of master.

File: templates/events/_event_header.html.twig [hunk 2]
```diff
@@ -17,7 +17,7 @@
             Ajouter au calendrier :
           </li>
           <li role="button" aria-label="{{ 'committee.event.show.share.cal_google'|trans }}">
-              <a href="https://calendar.google.com/calendar/render?action=TEMPLATE&amp;text={{ committee_event.name }}&amp;dates={{ committee_event.beginAt|date('Ymd') }}T{{ committee_event.beginAt|date('His') }}Z/{{ committee_event.finishAt|date('Ymd') }}T{{ committee_event.finishAt|date('His') }}Z&amp;location={{ committee_event.inlineFormattedAddress }}" target="_blank" title="{{ 'committee.event.show.share.cal_google'|trans }}">
+              <a href="https://calendar.google.com/calendar/render?action=TEMPLATE&amp;text={{ committee_event.name| url_encode }}&amp;dates={{ committee_event.beginAt|date('Ymd') }}T{{ committee_event.beginAt|date('His') }}Z/{{ committee_event.finishAt|date('Ymd') }}T{{ committee_event.finishAt|date('His') }}Z&amp;location={{ committee_event.inlineFormattedAddress }}" target="_blank" title="{{ 'committee.event.show.share.cal_google'|trans }}">
```
Comment (illusionOfParadise) [inline templates/events/_event_header.html.twig]: Extra space

**Comment by anthonykgross** [comment]
@illusionOfParadise Your requirements were changed ! Thanks for your advice 
Let me know if I can help you more

File: tests/Controller/EnMarche/EventControllerTest.php [hunk 3]
```diff
@@ -338,6 +338,20 @@ private function assertRedirectionEventNotPublishTest($url)
...
+    {
+        $this->authenticateAsAdherent($this->client, 'benjyd@aol.com', 'HipHipHip');
+
+        $event = $this->getEventRepository()->findOneByUuid(LoadEventData::EVENT_14_UUID);
+        $eventUrl = sprintf('/evenements/%s/%s/inscription', LoadEventData::EVENT_14_UUID, $event->getSlug());
+        $crawler = $this->client->request('GET', $eventUrl);
+
+        $this->assertResponseStatusCode(Response::HTTP_OK, $this->client->getResponse());
+        $link = $crawler->filter('.list__links.list__links--row.list__links--default li a')->eq(0)->attr('href');
+        $needle = http_build_query(['text' => $event->getName()], '', '&', PHP_QUERY_RFC3986);
```
Comment (Remg) [inline tests/Controller/EnMarche/EventControllerTest.php]: In functional tests it's always better to test expected values against hardcoded ones (not computed ones).

Prefer declaring something like `$needle = 'text=Meeting%20%2311%20de%20Brooklyn';` instead.

**Comment by anthonykgross** [comment]
Thanks for your sharing @Remg 
I applied your advice.

**Comment by marineb** [comment]
Thanks so much @anthonykgross! We'll be deploying this today. 🚀

**Pull request merged**